### ISSUE TITLE
NetCDF 1.4.0 causes python-aodndata to fail on Travis with "netcdftime" errors

### ISSUE THREAD

**Issue by ghost** [aodn/cc-plugin-imos#9]
@mhidas @lbesnard 

Looks like someone else hit the same problem, and solved it by pinning NetCDF to <1.4.0:

https://github.com/ioam/geoviews/commit/b5f1b799ddfd684245ca6b86e474f17bd0ee8fb4#diff-
2eeaed663bd0d25b7e608891384b7298

```
(travis-python-aodndata) 04:00:46 ~/virtualenvs/travis-python-aodndata/local/lib/python2.7/site-packages/cf_units$ compliance-checker -l
Traceback (most recent call last):
  File "/home/lwgordon/virtualenvs/travis-python-aodndata/bin/compliance-checker", line 7, in <module>
    from cchecker import main
  File "/home/lwgordon/virtualenvs/travis-python-aodndata/bin/cchecker.py", line 7, in <module>
    from compliance_checker.cf.util import download_cf_standard_name_table
  File "/home/lwgordon/virtualenvs/travis-python-aodndata/local/lib/python2.7/site-packages/compliance_checker/cf/__init__.py", line 1, in <module>
    from compliance_checker.cf.cf import (
  File "/home/lwgordon/virtualenvs/travis-python-aodndata/local/lib/python2.7/site-packages/compliance_checker/cf/cf.py", line 8, in <module>
    from compliance_checker.cf import util
  File "/home/lwgordon/virtualenvs/travis-python-aodndata/local/lib/python2.7/site-packages/compliance_checker/cf/util.py", line 10, in <module>
    from cf_units import Unit
  File "/home/lwgordon/virtualenvs/travis-python-aodndata/local/lib/python2.7/site-packages/cf_units/__init__.py", line 41, in <module>
    import netcdftime
ImportError: No module named netcdftime

```

I've confirmed that downgrading it in a virtualenv fixes the issue. We need to work out the appropriate solution for our situation.

**Comment by ghost** [comment]
https://github.com/ioos/compliance-checker/issues/579

I've logged it here because I believe the root cause issue lies with the dependency handling of the compliance-checker package.

I think given that, our solution in the short term should be to downgrade NetCDF4 back to the known good version and ensure Chef doesn't upgrade it to the latest (which is basically what it is doing now, and shouldn't be).

**Comment by mhidas** [comment]
Temporarily fixed in #12 and https://github.com/aodn/chef/pull/688.
We may as well keep this issue open until IOOS fix this properly in the core checker.

**Comment by ghost** [comment]
Closing this as it has been resolved both upstream and in-house
